Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -2,9 +2,45 @@ import macro from 'vtk.js/Sources/macros';
 import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
 import Constants from 'vtk.js/Sources/Rendering/Core/VolumeMapper/Constants';
 import vtkAbstractMapper from 'vtk.js/Sources/Rendering/Core/AbstractMapper';
+import vtkPiecewiseFunction from 'vtk.js/Sources/Common/DataModel/PiecewiseFunction';
+import vtkColorTransferFunction from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction';
 
 const { BlendMode, FilterMode } = Constants;
 
+function createRadonTransferFunctions(values) {
+  const ofun = vtkPiecewiseFunction.newInstance();
+  ofun.addPoint(-3024, 0.1);
+  ofun.addPoint(-637.62, 0.1);
+  ofun.addPoint(700, 0.5);
+  ofun.addPoint(3071, 0.9);
+
+  const ctfun = vtkColorTransferFunction.newInstance();
+  ctfun.addRGBPoint(-3024, 0.0, 0.0, 0.0);
```

Comment:
why -3024 and not -1024 ?

Reply:
In order to fit with the piecewise function.